### PR TITLE
Remove develop branches from test

### PR DESCRIPTION
**Pull request by spydon**
# Description

Since the develop branches don't exist anymore, remove the reference to them in the github workflow file.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

If something is unclear, please submit the PR anyways and ask about what you thought was unclear.

- [x] This branch is based on the latest `master`
- [x] I have added an entry under `[next]` in `CHANGELOG.md`
- [x] I have formatted my code with `flutter format`
- [x] I have made corresponding changes to the documentation
- [x] I have added examples for new features in `doc/examples`
- [x] The continuous integration (CI) is passing

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -4,9 +4,7 @@ on:
   push:
     branches:
       - master
-      - develop
       - master-v0.x
-      - develop-v0.x
```
Comment (erickzanardo) [inline .github/workflows/test.yml:9]: This branch I would keep, at least until we complete v1

**Review by erickzanardo — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (spydon) [reply .github/workflows/test.yml:9]: How come? If we need to patch `0.28` we can do it directly on `master-v0.x` right?

Comment (erickzanardo) [reply .github/workflows/test.yml:9]: Maybe we could drop that branch the ?

Comment (spydon) [reply .github/workflows/test.yml:9]: Honestly I thought we already had dropped it :smile:

**Pull request merged**